client/src/component/profilePicture/profilePicture.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import "./profilePicture.css";
+
+class ProfilePicture extends React.Component {
+  render() {
+    const { img_link } = this.props;
+
+    return <img class="profile-pic" src={img_link} />;
```

Comment:
Might need to change how we load up the src depending on where we get the images from. 

`src={require("../../assets/images/" + this.props.icon)}`